doc/source/installation.rst

```diff
@@ -115,15 +107,22 @@ Without the use of a container the installation of *ksrates* and its dependencie
         *   Or move ``codeml`` to another directory (here assumed to be ``~/bin``) and add this directory to ``$PATH``, for the Bash shell by copying the following line to the shell initialization file (e.g. ``.bashrc``)::
 
                 export PATH=$PATH:~/bin
-    
-    Please refer to PAML `website <http://abacus.gene.ucl.ac.uk/software/paml.html#download>`__ for more information about its installation.
 
-5. Install I-ADHoRe 3.0 from its GitHub `page <https://github.com/VIB-PSB/i-ADHoRe>`__ (required only for collinearity analysis of genome data).
+3. Install I-ADHoRe 3.0 from its GitHub `page <https://github.com/VIB-PSB/i-ADHoRe>`__ (required only for collinearity analysis of genome data).
+
+4.  Clone the *ksrates* repository from `GitHub <https://github.com/VIB-PSB/ksrates>`__ and install the package and its Python dependencies::
+
+        git clone https://github.com/VIB-PSB/ksrates
+    	cd ksrates
+    	pip3 install .
 
 
 Testing your installation
 =========================
 
+.. note::
+    WSL2 users can enter the Windows file system (e.g. ``Documents``) from the terminal through ``cd mnt/c/Users/your_username``.
```

Comment:
Can we assume it's always "c/Users/your_username" and that exists or shall we note it's an example?

Reply:
Yes, it's always with this path!

Reply:
What if a user has their data on drive D: or not under Users/?